tests/nnstreamer_decoder_image_segment/runTest.sh

```diff
@@ -78,6 +78,26 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! videoc
 
 callCompareTest test_golden.0 test_output.0 1 "test with videotestsrc" 0
 
+# THIS WON'T FAIL, BUT NOT MUCH MEANINGFUL (TARGET MODEL IS DIFFERENT ONE)
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
+videotestsrc num_buffers=1 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=257,height=257 ! tee name=t \
```

Comment:
usually ```num_buffers```  with 3 or more is better unless you really need to do it for 1 frame only.

Reply:
Right, I increase the number to 4. Thanks!